srcpkgs/wireproxy/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'wireproxy'
+pkgname=wireproxy
+version=1.0.1
+revision=0
+build_style=go
+go_import_path="github.com/octeep/wireproxy"
+go_package="${go_import_path}/cmd/wireproxy"
+short_desc="Wireguard client that exposes itself as a socks5 proxy"
+maintainer="Wind Wong <octeep@pm.me>"
+license="ISC"
+homepage="https://github.com/octeep/wireproxy"
+distfiles="https://github.com/octeep/wireproxy/archive/v${version}.tar.gz"
+checksum=e21eac22ef1b12dc2d7b3e5b58dcff183af9c1547be3e55a7e6c71394f053e1e
+
+system_accounts="wireproxy"
+caddy_homedir="/var/lib/wireproxy"
+
+conf_files="/etc/wireproxy/config"
+
+make_dirs="
+	/etc/wireproxy 0700 wireproxy wireproxy
+	/var/lib/wireproxy 0700 wireproxy wireproxy"
+
+post_install() {
+	vsv wireproxy
+
+	printf "\n" > config
+	vinstall config 644 etc/wireproxy
```

Comment:
use vconf

Reply:
Is there a way to make the config file only readable by the wireproxy user with vconf? The config file contains secret, which is why I made /etc/wireproxy/ owned by the wireproxy user in the first place.

Reply:
see other comment, also is there a reason to install a blank config?

Reply:
Is there a reason to install an empty config?

Reply:
> Is there a reason to install an empty config?

I just copied this behavior from caddy because that's what it seems to do. I guess it tells the user where the config file should be?

Reply:
Okay I've tested it. `conf_files` specifies that there should be a config file at `/etc/wireproxy/config`.  Creating an empty file solves that problem, otherwise this would happen:
```
=> ERROR: wireproxy-1.0.1_1: '/etc/wireproxy/config' configuration file not in PKGDESTDIR!

Reply:
conf_files shouldn't be needed if the conf file is not owned by the package.